presto-spi/src/main/java/com/facebook/presto/spi/StandardErrorCode.java

```diff
@@ -47,6 +47,7 @@
     SUBQUERY_MULTIPLE_ROWS(0x0000_001C),
     PROCEDURE_NOT_FOUND(0x0000_001D),
     INVALID_PROCEDURE_ARGUMENT(0x0000_001E),
+    NO_MATCHING_QUEUE(0x0000_001F),
```

Comment:
Since we're going to replace "queues" with "resource groups" I'd name this something else. Maybe "QUERY_REJECTED"?